### ISSUE TITLE
[Help request]: Translate PowerShell template to Batch

### ISSUE THREAD

**Issue by Griefed** [Griefed/ServerPackCreator#440]
### Whats the feature you would like to be improved?

This is open for anyone interested.

The task at hand is to convert the [PowerShell start-script template](https://github.com/Griefed/ServerPackCreator/blob/ms6/serverpackcreator-api/src/main/resources/de/griefed/resources/server_files/default_template.ps1) to Batch, as PowerShell continues to cause trouble on many systems.

Requirements:
- The Batch template must have excatly the same functionality as the PowerShell/bash template
- The Batch template must support all SPC-supported modloaders, just like the other templates
- The Batch template must contain all variables the PowerShell template has
- The Batch template must contain all placeholders the PowerShell template has
- The Batch template must source the values from the variables.txt generated alongside the scripts

In short: Translate the PowerShell template to Batch. Don't loose features, don't add features. 

### Anything else you would like to add?

If you feel up to the task, comment on this issue!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Griefed** [comment]
Unfortunately, I haven't had the time to tackle this issue myself.
Help from the community would be greatly appreciated.

**Comment by Griefed** [comment]
Better late than never.
This appears to work just fine, though I don't particularly like it:

```batch
@ECHO OFF
CD /d %~dp0
powershell.exe -ExecutionPolicy Bypass -File "start.ps1"
```

The reason for me not liking is is the fact this is more of a hack than a standalone Batch-template. It requires the PowerShell-script to be present and depends on PowerShell allowing this behaviour. 

I'll continue looking into providing an honest Batch-template, maybe even ask for the BisectHosting services on the server.

**Comment by kreezxil** [comment]
> Better late than never. This appears to work just fine, though I don't particularly like it:
> 
> ```batchfile
> @ECHO OFF
> CD /d %~dp0
> powershell.exe -ExecutionPolicy Bypass -File "start.ps1"
> ```
> 
> The reason for me not liking is is the fact this is more of a hack than a standalone Batch-template. It requires the PowerShell-script to be present and depends on PowerShell allowing this behaviour.
> 
> I'll continue looking into providing an honest Batch-template, maybe even ask for the BisectHosting services on the server.

You should, they would gladly grant you one. You're popular enough, I'd say go for a VPS.

**Comment by Griefed** [comment]
Maybe my phrasing was bad. I was talking about the service-request section in the Developer Market :D
